### DISCUSSION TITLE
Feature request: open file in text editor with "open" command

### DISCUSSION THREAD

**Discussion by sammonius** [community/community#40694] [Codespaces] [answered]
### Select Topic Area

Product Feedback

### Body

This was one of my favourite features when I was using AWS. I recently started using codespaces but the feature's gone and it takes me way more time to open a file. The way it worked is that you just typed ```open file.txt``` into the terminal and it would open the file. You could also create a file with ```touch file.txt && open file.txt```. Could this feature be added to codespaces?

**Comment by jfullstackdev** [comment]
is it not possible in terminal using Bash ? because Codespaces has the Bash terminal, 

as far as I can remember, I can open the file through that command then will be opened in Vim

**Comment by sammonius** [comment]
Yeah, but that opens it in the terminal. I mean it could be used to open a file in the VSCode editor.

**Comment by eljog** [comment, marked as answer]
@sammonius if you are using vscode client you could try `code` command to create/open text files in vscode editor.
Example: `code file.txt`

More details on this command can be found in vscode documentation [here](https://code.visualstudio.com/docs/editor/command-line#_opening-files-and-folders).
Please let me know if that helps

**Reply by Tomasgresl17** [reply]
Google play
